### DISCUSSION TITLE
How to make Show Notification Mantine clickable?

### DISCUSSION THREAD

**Discussion by mh3i** [mantinedev/mantine#5970] [Q&A] [answered]
hello I'm using this notification function from mantine for my nextjs webiste

notifications.show({ title: 'Default notification', message: 'Hey there, your code is awesome! 🤥', })

I want to it to be clickable, as I tried to add these props to make it clickable: notifications.show({ href:"/" Link:"/" title: 'Default notification', message: 'Hey there, your code is awesome! 🤥', })

https://mantine.dev/x/notifications/

It doesn't work please help .

**Comment by rtivital** [comment, marked as answer]
`notifications.show({ component: 'a', href:"https://your.link", title: 'Default notification', message: 'Hey there, your code is awesome! 🤥', })`

**Reply by mh3i** [reply]
it works thanks a lot!
